pkg/app/client.go

```diff
@@ -0,0 +1,12 @@
+package app
+
+import "os/exec"
+
+func GetCOEVersion() ([]byte, error) {
```

Comment:
COE is ContainerOrchestrationEngine.  I was thinking of first trying with the OpenShift client and if that fails try kubectl.  That way we can support both clients.